cmd/main.go

```diff
@@ -52,28 +53,44 @@ func main() {
 		command = reposerver.NewCommand()
 	case "argocd-cmp-server":
 		command = cmpserver.NewCommand()
-		isCLI = true
 	case "argocd-commit-server":
 		command = commitserver.NewCommand()
+		isArgocdCLI = true
```

Comment:
isArgocdCLI for commit-server looks weird to me? Are you sure?

Reply:
+1, I think we can drop this line

Reply:
Seems like a merge error with the cmp line above.